gf-button/README.md

```diff
@@ -34,81 +33,78 @@ GFButton(
 ),
 ```
 
-GF Buttons can be styled with several attributes to look a specific way. Each and every attribute is described below.
```

Comment:
I deleted this line because I think it's duplicated, it should go only below Properties title.